codecov.yml

```diff
@@ -28,22 +28,22 @@ coverage:
     changes: no
 
 ignore:
-  #  - "*/interface/thrift/**/*.java"  # ignore thrift-gen folders and all its contents
-  - "**/generated-classes/**/*.java"
-  - "**/generated-sources/**/*.java"
-  - "grafana/**/*.java"
-  - "spark/**/*.java"
-  - "hadoop/**/*.java"
-  - "**/pom.xml"
-  - "**/*.md"
-  - "**/*.sh"
-  - "**/*.cmd"
-  - "**/*.bat"
-  - "**/*.yml"
-  - ".mvn"
-  - "mvnw"
-  - "NOTICE"
-  - "NOTICE-binary"
-  - "License"
-  - "LICENSE-binary"
-  - "DISCLAIMER"
+#  - "*/interface/thrift/**/*.java"  # ignore thrift-gen folders and all its contents
+- "**/generated-classes/**/*.java"
+- "**/generated-sources/**/*.java"
+- "grafana/**/*.java"
+- "spark/**/*.java"
+- "hadoop/**/*.java"
+- "**/pom.xml"
+- "**/*.md"
+- "**/*.sh"
+- "**/*.cmd"
+- "**/*.bat"
+- "**/*.yml"
+- ".mvn"
+- "mvnw"
+- "NOTICE"
+- "NOTICE-binary"
+- "License"
+- "LICENSE-binary"
+- "DISCLAIMER"
```

Comment:
I'm not sure whether the codecov will work if you change the format of this file. Better to keep the original format....